auth/ante_test.go

```diff
@@ -106,8 +105,8 @@ func TestNoSigs(t *testing.T) {
 	handler := NewAnteHandler(mapper, txIndex, feeAmount)
 	_, res, abort := handler(ctx, tx)
 
-	assert.Equal(t, true, abort, "did not abort with no signatures")
-	assert.Equal(t, sdk.ToABCICode(sdk.CodespaceType(1), sdk.CodeType(4)), res.Code, "tx had processed with no signatures")
+	require.Equal(t, true, abort, "did not abort with no signatures")
```

Comment:
I think the checks on the value of abort should stay asserts, since it can be useful to know what error number was returned in comparison to the ones expected.

Reply:
good point, will change back